src/main/java/gmail/mishchuk/lesson40_FI/LogicToFI.java

```diff
@@ -0,0 +1,56 @@
+package gmail.mishchuk.lesson40_FI;
+
+import java.util.Random;
+
+public class LogicToFI {
+
+//    public static boolean getPrimeNumberElvisStyle (int num) {
+//        if (num <= 1 ? false);
+//        for (int i = 2; i * i <= num; i++) {
+//            if (num % i == 0 ? false : true);
+//            }
+//        return true;
+//    }
+
+    public static boolean getPrimeNumber(int numberToCheck) {
+        if (numberToCheck <= 1) {
+            return false;
+        }
+        for (int i = 2; i * i <= numberToCheck; i++) {
+            if (numberToCheck % i == 0) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static int getGeneratedNumber(int maxNumber) {
+        Random random = new Random();
+        return random.nextInt(maxNumber + 1);
+    }
+
+    public static int getRandomDayOfWeek() {
+        return (int) (Math.random() * 8);
+    }
```

Comment:
Тут я хочу щоб повертився випадковий день тижня як стрінга 😉 

```suggestion
    public static String getRandomDayOfWeek() {
        // TODO (Понеділок / Вівторок / Середа ...)
    }
```

Reply:
там інт, бо використав світч (проходили на початку курсу)

Reply:
воно по факту повертає стрінгу, переписувати?

Reply:
Зробити через світч також можна (хоча і не так оптимально). Головне додайте лямбди в клас (у Вас одна й та сама помилка у всіх методах яку я [тут](https://github.com/VolodymyrMishchuk/homework-java/pull/27?authuser=0#discussion_r2054685185) згадував)